lti_emailer/settings/local.py

```diff
@@ -15,3 +15,14 @@
 }
 
 dictConfig(LOGGING)
+
+CANVAS_URL = 'http://canvas.icommons.harvard.edu/'
```

Comment:
@hktest and @Joy603 agreed, it's not black-and-white. I'm ok with reusing as much of the existing base settings as possible -- hence CANVAS_URL coming from secure.py -- but that the tests may have to define their own e.g. relative URLs for now. This goes back to trysting our data is in the same state every time we run a test... I think for now that's our only option. :) Technically the CANVAS_URL probably doesn't _need_ to be in the secure settings, just the base settings, but because it changes in different environments and AWS loads the secure.py settings that's where we pull it from, for convenience. Usernames/passwords/credentials and internal URLs like the selenium grid URL are naturals for including in secure.py, but public URLs probably don't need to be secure (though the argument could be made to put them in a settings file instead of 'baked into' the test files... I just think that's difficult right now given the state of our system and the overhead with making sure everyone's settings point to the right thing). Hope that helps.